shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/SelectorMapper.java

```diff
@@ -88,9 +88,9 @@ public interface SelectorMapper extends ExistProvider {
      * select select by name.
      *
      * @param name the name
-     * @return selector do
+     * @return selector do list
      */
-    SelectorDO selectByName(String name);
+    List<SelectorDO> selectByName(String name);
```

Comment:
new method do this? selectListByName or findByName?

Reply:
Yes. The selectByName method is currently only used in one place. I plan to remove it later because the original method may generate errors during use (using 'contextpath' as a conditional query usually results in 2 records, one from the proxy proxy plugin and the other from the contextpath plugin). This PR only quickly fixes discovered bugs.

Reply:
use a new method instead of changing the method

Reply:
> use a new method instead of changing the method

That's right.